start_esb-camel_docker.sh

```diff
@@ -0,0 +1,82 @@
+﻿#!/bin/bash
+set -e
+
+rabbitmq_host=${RABBITMQ_HOST:-localhost}
+rabbitmq_port=${RABBITMQ_PORT:-5672}
+rabbitmq_user=${RABBITMQ_USER:-guest}
+rabbitmq_password=${RABBITMQ_PASSWORD:-$(echo $secret_rabbitmq_password)}
+
+debug_port=${DEBUG_PORT:-8792}
+debug_suspend=${DEBUG_SUSPEND:-n}
+debug_print_bash_cmds=${DEBUG_PRINT_BASH_CMDS:-n}
+admin_url=${METASFRESH_ADMIN_URL:-NONE}
+server_port=${SERVER_PORT:-8184}
+
+
+echo_variable_values()
+{
+ echo "*************************************************************"
+ echo "Display the variable values we run with"
+ echo "*************************************************************"
+ echo "Note: all these variables can be set using the -e parameter."
+ echo ""
+ echo "DEBUG_PORT=${debug_port}"
+ echo "DEBUG_SUSPEND=${debug_suspend}"
+ echo "DEBUG_PRINT_BASH_CMDS=${debug_print_bash_cmds}"
+ echo "METASFRESH_ADMIN_URL=${admin_url}"
+ echo "SERVER_PORT=${server_port}"
+ echo ""
+ echo "RABBITMQ_HOST=${rabbitmq_host}"
+ echo "RABBITMQ_PORT=${rabbitmq_port}"
+ echo "RABBITMQ_USER=${rabbitmq_user}"
+ echo "RABBITMQ_PASSWORD=*******"
+ echo ""
+}
+
+# Note: the Djava.security.egd param is supposed to let tomcat start quicker, see https://spring.io/guides/gs/spring-boot-docker/
+run_metasfresh()
+{
+ if [ "$admin_url" != "NONE" ]; 
+ then
+	# see https://codecentric.github.io/spring-boot-admin/1.5.0/#spring-boot-admin-client
+	# spring.boot.admin.client.prefer-ip=true because within docker, the hostname is no help
+	metasfresh_admin_params="-Dspring.boot.admin.url=${admin_url} -Dmanagement.security.enabled=false -Dspring.boot.admin.client.prefer-ip=true"
+ else
+	metasfresh_admin_params=""
+ fi
+
+# thx to 
+# https://blog.csanchez.org/2017/05/31/running-a-jvm-in-a-container-without-getting-killed/
+MEMORY_PARAMS="-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=1"
```

Comment:
We had this for a brief period of time, but it doesn't work like that;
Docker will most likely kill this container when it comes under stress.
Pls contact julian for details. As it is (with java-8) our current approach is that instead of this, we use -Xmx with a value that is set from outside via ENV